index.html

```diff
@@ -121,6 +121,36 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
 
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
+                <!-- ================================================ -->
+        <!-- ==================  TEMPLATE  ================== -->
+        <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
+
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">compudefierro</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
+            <a href="https://www.twitter.com/your_user_handle" target="_blank">Your handle</a>
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com/your_user_handle" target="_blank">Your handle</a>
+          </p>
+          <p class="about">I'm new on this, I love Git and this good first issue is the best and better.</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://github.com/compudefierro/" target="_blank" title="First Resource">Compudefierro</a>
+              </li>
+              <li>
+                <a href="https://www.linkedin.com/in/jorge-osvaldo-dri-90602542/" target="_blank" title="Second Resource">Linkedin</a>
+              </li>
+              <li>
```

Comment:
As these two links that you have provided is your profile links, and not a helpful resource links for any sort of dev or coding related, please do remove them and provide some other resource links. 
Note that you are free to provide these links inside the `contact` section just few lines above :)